### ISSUE TITLE
Update to Webpack 2

### ISSUE THREAD

**Issue by timrijkse** [Blocklevel/blue#35]
Should we already?

**Comment by MatteoGabriele** [comment]
Still early for me. I've read few articles about it and I've tried it.
The only advantage we could have now is the tree shaking: which is really nice, but documentation is still lacking.
Let's wait the official release and check bigger community reaction

**Comment by ThaNarie** [comment]
I think we should now? It's already at 2.2, and the 2.x is 'latest' for over a week now.
We're using it in your very big and complex project as well, which works fine!

It basically all depends on the loaders, if they have added support for 2.x.
Could never hurt to create a PR to see if it works, and keep that up-to-date with master until it's ready to merge :)

**Comment by jonaskuiler** [comment]
You're right, I will work on this and see if it works. Because we did several projects we've learned some new approaches as well so I'll discuss this with @MatteoGabriele

**Comment by mientjan** [comment]
Already moving our projects to 2.2, will give you a update of our learnings
